### DISCUSSION TITLE
How do I update using Unraid Community Applications?

### DISCUSSION THREAD

**Discussion by slopokdave** [immich-app/immich#6495] [Q&A] [answered]
My apologies if this is redundant, but I can’t figure this out. I understand that Unraid docker will say “Update Available”, but it does not update. The documentation for Immich refers to updating the docker stack; but using community applications, a docker compose stack is not used.  Thanks in advance.

**Comment by martabal** [comment]
Currently, the latest version for the AIO image (the image used for the Community Applications) is `v1.92.1` https://github.com/imagegenius/docker-immich

**Reply by slopokdave** [reply]
Oh I understand now, so I need to wait for them to update, or move to the regular docker version. Got it. :) Thank you!

**Reply by martabal** [reply, marked as answer]
Exactly !

Usually the AIO image is updated a few minutes after the "official" release. But this time there are breaking changes in the Dockerfile and the PR that fixes them needs to be merged. This happens less and less often but still :)
